go.mod

```diff
@@ -75,4 +79,7 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace golang.org/x/net => golang.org/x/net v0.24.0
+replace (
+	github.com/open-feature/open-feature-operator/apis => github.com/odubajDT/open-feature-operator/apis v0.0.0-20240426164950-7aa22f92ccec
```

Comment:
i guess this can be removed

Reply:
this PR should not be merged, it is only for reviews (to this replace is OK)